src/web/components/Onwards/Carousel/Carousel.tsx

```diff
@@ -77,7 +72,7 @@ const carouselStyle = css`
 
 	position: relative; /* must set position for offset(Left) calculations of children to be relative to this box */
 
-	overflow-x: scroll; /* Scrollbar is less intrusive visually on non-desktop devices typically */
+	overflow-x: auto; /* Scrollbar is less intrusive visually on non-desktop devices typically */
```

Comment:
Not a major concern but I think we switched to `auto` when playing with this but I'm not sure if this is preferable over `scroll`. I read the docs on auto and it says

> Depends on the user agent. If content fits inside the padding box, it looks the same as visible, but still establishes a new block-formatting context. Desktop browsers provide scrollbars if content overflows.

So, yeah, I'm not sure I'm any wiser after reading that!

Reply:
I think that (based on a StackOverflow dive) the difference is that 
> Auto will only show a scrollbar when any content is clipped.
> Scroll will however always show the scrollbar even if all content fits and you cant scroll it.

So for our use case it should probably be `auto` even if we expect the content to always clip. If there is a situation where there's a single card or something, we don't allow scrolling and so we should reduce the risk of errors in the progress indicator.